molecules/_tables.scss

```diff
@@ -20,7 +20,7 @@ $table-row-border-color: darken($catkin, 5%);
     position: absolute;
     cursor: pointer;
     &.icon-desc {
-      top: rem(13px);
+      top: rem(16px);
```

Comment:
I think there is a variable for this ? e.g. $base-font-size is 1rem // 16px ? https://github.com/wholesale-design-system/scss/blob/master/utils/variables/_basics.scss#L8 

This would make it easier in the future if we might switch the font-size :)